index.js

```diff
@@ -60,10 +60,12 @@ module.exports = function(config) {
         logger = makeLogger(name, name);
         context.logger = logger;
         var getConnection = require('./lib/api/get_connection')(context);
+        var events = require('./lib/api/events');
         context.foundation = {
             makeLogger: makeLogger,
             startWorkers: require('./lib/api/start_workers')(context),
-            getConnection: getConnection
+            getConnection: getConnection,
+            events: events
```

Comment:
I tried to consider the alternative at sticking it at the top level of context itself. It seemed to bolted on and did not seem to have any notion of it being a part of terafoundation itself. Would you prefer that there was some function that would set and listen to events instead of the actual event emitter?